### DISCUSSION TITLE
How to put an attachment into S3 Storage or any cloud service else ?

### DISCUSSION THREAD

**Discussion by hblab-hautp** [Budibase/budibase#3838] [Help] [answered]
Hi team, hi everyone

I'm trying to figure out how to push my attachment (file) into another storage service that is not Minio local, but i seem have not anyway to do it.
Please tell me the right way if it possible !

Thank you very much !

**Comment by mjashanks** [comment, marked as answer]
It's not possible right now, but it should be in our REST API updates - due in January.

**Reply by hblab-hautp** [reply]
thank for your anwser. I'm waitting for it
